checks/libs/win/pdhbasecheck.py

```diff
@@ -0,0 +1,64 @@
+# (C) Datadog, Inc. 2013-2016
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+# project
+from checks import AgentCheck
+from utils.containers import hash_mutable
+# datadog
+try:
+    from checks.libs.win.winpdh import WinPDHCounter
+except ImportError:
+    def WinPDHCounter(*args, **kwargs):
```

Comment:
this should be a class instead of a function? You'd probably want to have the check error out if it can't import it, but not on import. That will be ugly